src/request.js

```diff
@@ -0,0 +1,39 @@
+const createGame = () => {
+  const games = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+  return fetch(games, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'Montanholi\'s Game',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((json) => json.result.split(' ')[3]);
+};
```

Comment:
- [ ] Please Use `async` and `await` with the fetch method to send and receive data.